redun/executors/aws_batch.py

```diff
@@ -852,7 +852,7 @@ def __init__(
         self.default_task_options: Dict[str, Any] = {
             "vcpus": config.getint("vcpus", fallback=1),
             "gpus": config.getint("gpus", fallback=0),
-            "memory": config.getint("memory", fallback=4),
```

Comment:
This change could be (not asking though) moved to its own PR since it's really a bugfix for aws_batch.
That way if it causes problems it can be rolled back independently.